lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -215,6 +219,8 @@ sub limit_assets {
     if (opendir($dh, $OpenQA::Utils::assetdir . "/repo")) {
         my %repos;
         while (readdir($dh)) {
+            next if $_ eq '.' || $_ eq '..';
```

Comment:
IMHO either explicit variable names should be used, i.e. not '$_' but a better name (would 'while $file = readdir($dh)' work?), or just go implicit as aaannz proposed and don't mention '$_' at all.